iocore/cluster/P_ClusterCacheInternal.h

```diff
@@ -329,14 +330,8 @@ struct CacheContinuation : public Continuation {
   inline static bool
   is_ClusterThread(EThread *et)
   {
-    int etype = ET_CLUSTER;
-    int i;
-    for (i = 0; i < eventProcessor.n_threads_for_type[etype]; ++i) {
-      if (et == eventProcessor.eventthread[etype][i]) {
-        return true;
-      }
-    }
-    return false;
+    auto &tg = eventProcessor.thread_group[ET_CLUSTER];
+    return ET_CLUSTER && std::any_of(tg._thread, tg._thread + tg._count, [et](EThread *t) { return t == et; });
```

Comment:
That's how the current code works. The type is a bit mask. But more realistically, it's cluster code, it should never be called. But I might want to put the `ET_CLUSTER` check first - if it's zero then the cluster group was never set up.